types/mui-datatables/index.d.ts

```diff
@@ -194,6 +194,10 @@ export interface MUIDataTableOptions {
     filter?: boolean;
     filterType?: FilterType;
     fixedHeader?: boolean;
+    fixedHeaderOptions?: {
```

Comment:
> [ x] Add or edit tests to reflect the change. (Run with npm test.)

Can we amend the `mui-datatables-tests.tsx` to cover that change?
Just like in:
https://github.com/gregnb/mui-datatables/blob/8188cafd37e6ed76876c8129b5e928b18c73ba4d/examples/fixed-header/index.js#L82-L85
so,
```tsx
        sortFilterList: false,
        fixedHeaderOptions: {
            xAxis: false,
            yAxis: true
        },
        customRowRender: (data, dataIndex, rowIndex) => {
```

Reply:
will do